ci/build_common.sh

```diff
@@ -167,7 +167,7 @@ function configure_preset()
     local GROUP_NAME="🛠️  CMake Configure ${BUILD_NAME}"
 
     pushd .. > /dev/null
-    run_command "$GROUP_NAME" cmake --preset=$PRESET --log-level=VERBOSE "${GLOBAL_CMAKE_OPTIONS[@]}" $CMAKE_OPTIONS
+    run_command "$GROUP_NAME" cmake --preset=$PRESET --log-level=VERBOSE $CMAKE_OPTIONS "${GLOBAL_CMAKE_OPTIONS[@]}"
```

Comment:
I believe we need to adopt the line
> echo " -cmake-options: Additional options to pass to CMake"
to tell the users what it does now

Reply:
The behavior still matches the docs. The only difference is how user-supplied options are ordered around hardcoded options in the build scripts.